ebu_tt_live/xsd/ebutt_metadata.xsd

```diff
@@ -421,13 +412,23 @@ Please note that the EBU-TT XML Schema is a helping document and NOT normative b
 						</xs:documentation>
 					</xs:annotation>
 				</xs:element>
-			    <xs:element name="documentFacet" type="ebuttm:documentFacet_type" minOccurs="0" maxOccurs="unbounded"/>
-			    <xs:element name="trace" type="ebuttm:trace_type" minOccurs="0" maxOccurs="unbounded"/>
+				<xs:element name="documentFacet" type="ebuttm:documentFacet_type" minOccurs="0"/>
+				<xs:element name="appliedProcessing" minOccurs="0">
+					<xs:complexType mixed="true">
```

Comment:
Would be good to add an annotation for this, summarising what is in the element in ebu-tt part m.